distribution/packages/src/rpm/wazuh-indexer.rpm.spec

```diff
@@ -158,18 +159,30 @@ chmod -Rf a+rX,u+w,g-w,o-w %{buildroot}/*
 exit 0
 
 %pre
-set -e
-# Stop existing service
-if command -v systemctl >/dev/null && systemctl is-active %{name}.service >/dev/null; then
-    echo "Stop existing %{name}.service"
-    systemctl --no-reload stop %{name}.service
-    mkdir -p %{tmp_dir}
-    touch %{tmp_dir}/wazuh-indexer.restart
+# Remove -x once solution is approved
+set -ex 
+
+# Only during upgrade
+if [ "$1" -gt 1 ]; then
+    echo "Running upgrade pre-script"
+
+    # Mark if service is running
+    if command -v systemctl >/dev/null && systemctl is-active --quiet %{name}.service; then
+        echo "Service is active, marking it for restart"
+        touch %{state_file}
```

Comment:
The DEB package is still using the previous "state" file (`%{tmp_dir}/wazuh-indexer.restart`) shouldn't it also be updated? It would be better if both uses the same file naming and location

Reply:
Added in [9439417](https://github.com/wazuh/wazuh-indexer/pull/794/commits/9439417a66ee687034d42d2fb2fba71f88b9a283)